README.md

```diff
@@ -86,6 +87,41 @@ As a modern Python-based environment, GamestonkTerminal opens access to numerous
 
 ## Getting Started
 
+### Docker Installation - *new and improved*
+
+0. First step is to make sure docker desktop is installed.  Install links can be found [here](https://www.docker.com/products/docker-desktop).  To confirm that your docker desktop is downloaded and running, open a command prompt or terminal and enter
+`docker info`.  If you get the following you are not running the docker desktop:
+```bash
+Server:
+ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
+```
+Open the docker desktop app in this case.
+
+1. Download the latest docker image.
+
+```bash
+docker pull ghcr.io/gamestonkterminal/gst-poetry:latest
+```
+
+Upon running this the first time, you should see the various layers downloading (note the random series of letters numbers will vary).
```

Comment:
I would also add that the first download will take time, but the subsequent downloads will be significantly faster.